diff -Naupr ipvsadm-1.26.orig/ipvsadm.c ipvsadm-1.26/ipvsadm.c
--- ipvsadm-1.26.orig/ipvsadm.c	2013-07-26 11:46:17.780747811 -0500
+++ ipvsadm-1.26/ipvsadm.c	2013-07-26 12:42:45.529737732 -0500
@@ -424,6 +424,7 @@ parse_options(int argc, char **argv, str
 		{ "exact", 'X', POPT_ARG_NONE, NULL, 'X', NULL, NULL },
 		{ "ipv6", '6', POPT_ARG_NONE, NULL, '6', NULL, NULL },
 		{ "ops", 'o', POPT_ARG_NONE, NULL, 'o', NULL, NULL },
+		{ "ops", 'O', POPT_ARG_NONE, NULL, 'O', NULL, NULL },
 		{ "pe", '\0', POPT_ARG_STRING, &optarg, TAG_PERSISTENCE_ENGINE,
 		  NULL, NULL },
 		{ NULL, 0, 0, NULL, 0, NULL, NULL }
@@ -648,6 +649,7 @@ parse_options(int argc, char **argv, str
 				fail(2, "-6 used before -f\n");
 			}
 			break;
+		case 'O':
 		case 'o':
 			set_option(options, OPT_ONEPACKET);
 			ce->svc.flags |= IP_VS_SVC_F_ONEPACKET;
